index.js

```diff
@@ -1,18 +1,87 @@
 /***** Deliverable 1 *****/
+
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
+header.style.color = "cyan"
+header.style.backgroundColor = "black"
+header.style.border = "chartreuse"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+
+
+console.log('This is what the traveler object looks like: ', traveler)
+
+const profile = document.querySelector("div#profile")
+profile.innerHTML = `<div id="profile"> 
+
+    <img src="${traveler.photo}"
+      
+    alt="${traveler.name}">
+    
+    <h2>${traveler.name}</h2>
+
+    <em>${traveler.nickname}</em>
+
+    </div>
+
+    <h4>Great Sightings</h4>
+
+    <ul id="animals">
+    <!-- add an <li> for each animal sighting here -->
+    </ul>`
```

Comment:
This works! However, take note that if this code is written after deliverable 4, it would undo deliverable 4's functionality. Also, if we had any references to any of these elements before this deliverable, by using innerHTML we would lose those references. To make the code more scalable in this case, it may be better to search for the elements on the DOM using querySelector since they already exist, store each in a variable, and update the necessary properties of each.